commands/configurations_test.go

```diff
@@ -15,3 +15,202 @@ limitations under the License.
 */
 
 package commands_test
+
+import (
+	"github.com/redhatinsighs/insights-operator-cli/commands"
+	"github.com/tisnik/go-capture"
+	"strings"
+	"testing"
+)
+
+func tryToFindConfiguration(t *testing.T, captured string, configuration string) {
+	if !strings.Contains(captured, configuration) {
+		t.Fatal("Can not find configuration:", configuration)
+	}
+}
+
+// TestListOfConfigurations checks whether the non-empty list of configurations read via REST API is displayed correctly
+func TestListOfConfigurations(t *testing.T) {
+	configureColorizer()
+	restAPIMock := RestAPIMock{}
+
+	captured, err := capture.StandardOutput(func() {
+		commands.ListOfConfigurations(restAPIMock, "")
+	})
+
+	checkCapturedOutput(t, captured, err)
+	if !strings.HasPrefix(captured, "List of configurations for all clusters") {
+		t.Fatal("Unexpected output:\n", captured)
+	}
+
+	numlines := strings.Count(captured, "\n")
+	if numlines <= 4 {
```

Comment:
maybe some comment about why 4 would be nice :)

Reply:
Good catch. I've added some comments in the last commit (below)